lang/en/facetoface.php

```diff
@@ -363,6 +364,8 @@
 $string['setting:addchangemanageremail_caption'] = 'Manager\'s email:';
 $string['setting:attendeesexportfields_caption'] = 'Attendees export fields';
 $string['setting:attendeesexportfields'] = 'Select the fields to be included in a session\'s exported list of attendees. This will be in addition to the attendee\'s first and last name.';
+$string['setting:enableapprovals'] = 'When disabled, approvals can not be required for any Face-to-Face activity.';
```

Comment:
Cannot -> are not?

Reply:
Incorrect. They do not mean the same thing. When disabled, the option to add approvals via the activity setting is no longer available.

Reply:
No action taken.

Reply:
```suggestion
$string['setting:enableapprovals'] = 'When disabled, the option to add approvals via the activity setting is no longer available';
```
That's much clearer for me.

Reply:
updated